src/exo6 - ReaderTaskEither/exo6.solution.ts

```diff
@@ -102,13 +102,13 @@ export const getConcatenationOfTheTwoUserNamesUsingAp = ({
 // and bind the return value to feed the context and use this data.
 
 export const getConcatenationOfTheBestFriendNameAndUserName = ({
-  userIdOne,
+  userId,
 }: {
-  userIdOne: string;
+  userId: string;
 }) =>
   pipe(
     rte.Do,
-    rte.apS('userOne', User.Repository.getById(userIdOne)),
+    rte.apS('userOne', User.Repository.getById(userId)),
```

Comment:
What could be better here would be to name it `user` and the other one `bestFriend` as you did in your other PR as it's clearer than a generic userOne/userTwo :)